karmagrambot/__init__.py

```diff
@@ -30,9 +36,9 @@ def save_message(message, db):
         length = len(message.caption)
 
     vote = None
-    if message.text == '+':
+    if check_message(message.text):
         vote = '+'
-    elif message.text == '-':
+    elif:
```

Comment:
`elif` with no condition has no meaning and isn't even syntactically valid. Did you either run this code or a similar, minimum version?